src/plugins/visualizations/public/visualize_app/app.tsx

```diff
@@ -45,10 +46,24 @@ const NoDataComponent = ({
   dataViewEditor,
   onDataViewCreated,
 }: NoDataComponentProps) => {
+  const hasCustomBranding = core.customBranding.hasCustomBranding$.pipe(() => {
```

Comment:
We should be subscribing to the observable in our component, not in the consuming plugins.

Reply:
I think this is where I might be stuck.